src/pudl/metadata/resources/ferc1.py

```diff
@@ -823,6 +823,22 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "denorm_plants_utils_ferc1": {
+        "description": "Denormalized table that contains FERC plant and utility information.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "plant_name_ferc1",
+                "plant_id_pudl",
+                "utility_name_ferc1",
+                "utility_id_pudl",
+            ],
+        },
+        "field_namespace": "ferc1",
+        "etl_group": "outputs",
+        "sources": ["ferc1"],
+        "include_in_database": False,
```

Comment:
And we're not including this in the DB because it's just an intermediate table that gets joined onto other denormalized tables?

Reply:
We are not including it in the database because it is a view.

Reply:
Ah okay so this is like "don't include in database metadata" -- it's in the DB, but we're not creating the table like we do for Python outputs.

Reply:
We should change this Resource attribute.

Reply:
I renamed this to `create_database_schema`/